reference/5.1/Microsoft.PowerShell.Core/Get-History.md

```diff
@@ -21,80 +21,83 @@ Get-History [[-Id] <Int64[]>] [[-Count] <Int32>] [<CommonParameters>]
 
 ## DESCRIPTION
 
-The **Get-History** cmdlet gets the session history, that is, the list of commands entered during the current session.
+The `Get-History` cmdlet gets the session history, that is, the list of commands entered during the
+current session.
 
-PowerShell automatically maintains a history of each session.
-The number of entries in the session history is determined by the value of the $MaximumHistoryCount preference variable.
-Beginning in Windows PowerShell 3.0, the default value is 4096.
+PowerShell automatically maintains a history of each session. The number of entries in the session
+history is determined by the value of the `$MaximumHistoryCount` preference variable. Beginning in
+Windows PowerShell 3.0, the default value is **4096**. By default, history files are saved in the
```

Comment:
I think values should be wrapped in backticks?

Reply:
fixed